.changelog/f4263acfa9964f27a95087734e6aed0d.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "f4263acf-a996-4f27-a950-87734e6aed0d",
+    "type": "feature",
+    "description": "Updated service model to latest version.",
```

Comment:
I see multiple .changelog file  for updating service model to latest version.  Intentional? 

I see `service/ssm` in both.

Reply:
Good eye, SSM should not be in both. The intention was to break out the documentation changes (patch bumps) from feature changes (minor bumps). Seems like there is a bug in this case in the git detection logic, as `ssm` should be marked for a feature bump and `ssmcontacts` should be just a patch bump. My likely hunch is this is getting incorrectly detected in both because of both having the same matching prefix. Will create a backlog item to investigate. This won't block the release though.